examples/routing_service/routing_service_constrained_wan/wandemo/local.tcp.transport.xml

```diff
@@ -0,0 +1,258 @@
+<?xml version="1.0"?>
+<!--
+ (c) 2005-2020 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+ RTI grants Licensee a license to use, modify, compile, and create derivative
+ works of the Software.  Licensee has the right to distribute object form only
+ for use with RTI products.  The Software is provided "as is", with no warranty
+ of any type, including any warranty for fitness for any purpose. RTI is under
+ no obligation to maintain or support the Software.  RTI shall not be liable for
+ any incidental or consequential damages arising out of the use or inability to
+ use the software.
+ -->
+<dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+     xsi:noNamespaceSchemaLocation="https://community.rti.com/schema/6.0.0/rti_routing_service.xsd">
+    <!-- ********************************************************************** -->
+    <!-- RTI Routing service examples for TCP over WAN                          -->
+    <!-- Note: Change the IPs and ports below to fit yours                      -->
+    <!-- ********************************************************************** -->
+    <!-- UDP/Shared memory (domain 0) => TCP (domain 1)                         -->
+    <!-- ********************************************************************** -->
+    <routing_service name="example">
+
+        <annotation><documentation>In a WAN, routes from UDP or shared memory to TCP. Remember to edit the file and change the public IP address.</documentation></annotation>
+        <administration><domain_id>0</domain_id></administration>
+
+        <domain_route name="Route">
+            <participant name="participant0">
+                <domain_id>0</domain_id>
+            </participant>
+
+            <participant name="participant1">
+                <domain_id>1</domain_id>
+                <participant_qos>
+                    <discovery>
+                        <initial_peers>
+                            <element>tcpv4_wan://50.18.214.8:7400</element>
+                        </initial_peers>
+                    </discovery>
+                    <discovery_config>
+                        <participant_liveliness_assert_period>
+                            <sec>10</sec>
+                            <nanosec>0</nanosec>
+                        </participant_liveliness_assert_period>
+                    </discovery_config>
+                    <transport_builtin>
+                        <mask>MASK_NONE</mask>
+                    </transport_builtin>
+                    <property>
+                        <value>
+                            <element>
+                                <name>dds.transport.load_plugins</name>
```

Comment:
If we are using 6.0.1 we can use the builtin QoS Snippet for TCP:
* Transport.TCP.LAN.Client
* Transport.TCP.WAN.Symmetric.Client
* Transport.TCP.WAN.Asymmetric.Client
* Transport.TCP.WAN.Asymmetric.Server

Reply:
Unless you strongly disagree, I think I'd rather spell out all the TCP options here. This may help customers that are tweaking their example setup to adjust some of these settings (versus tracking down the snippet and modifying that directly).

But if you'd like, I could add a comment about how snippets could be used instead of configuring the tcp settings this way